### ISSUE TITLE
Properties with empty value ale not filtered out

### ISSUE THREAD

**Issue by sqal** [emotion-js/emotion#2230]
**Current behavior:**

Here's the problem. Currently, when interpolation evaluates to falsy value (undefined, null, false or ""), emotion will render that property with an empty string. This property will of course be ignored by the browser but shouldn't it be removed from the final css result?

![screenshot](https://i.imgur.com/kcAizL3.png)

**To reproduce:**

```js
const Container = styled.div`
  padding: 2rem;
  color: blue;
  background: ${undefined};
`;

<div css={css`
  padding: 2rem;
  color: blue;
  background: ${props => props.someMaybeFalsyValue};
`} />
```

[Demo on codesandbox](https://codesandbox.io/s/keen-snyder-hezr9?fi)

**Expected behavior:**

Emotion should correctly handle iterpolation results and don't render properties with invalid value in the final css.
Is it a bug or a breaking change in v11, because it worked fine in v10?

**Environment information:**

- `react`: 17.0.1
- `@emotion/react`: 11.1.4
- `@emotion/styled`: 11.1.4

**Comment by Andarist** [comment]
It's probably a duplicate of https://github.com/emotion-js/emotion/issues/2162 and the issue was fixed by https://github.com/thysultan/stylis.js/pull/251 but a new release has not been published. Could you create a release request in https://github.com/thysultan/stylis.js ?

**Comment by sqal** [comment]
Yup, that's the same issue. Sorry for the duplicate. I asked the author for version bump

**Comment by sqal** [comment]
@Andarist stylis author released the fix in 4.0.7 ;)

**Comment by Andarist** [comment]
I'll update the Stylis version when I get a chance to (although I could use a PR for this). That being said - you should be able to update it in your project even without that as we are using `^` range to define this dependency and this should allow you to install the new version of Stylis.
